LISISO/commonfunctions.sh

```diff
@@ -315,3 +315,54 @@ function checkrpms()
 		echo "RPM's are missing"
 	fi
 }
+
+function CheckPartitionSize()
+{
+        if (( $1 < $2 ));then
+                echo "LIS installation aborted due to insufficiant space"
+                printf "\t %s \t\t %s\n" "$3" "available: $1 required: $2 Bytes"
+                return 0
+        fi
+        return 1
+}
+
+function CheckRequiredSpace()
+{
+	No_of_symvers=$(($(ls /boot | grep "symvers" | wc -l)))
+	# 160MB(167772160 Bytes) minimum space is required for 1 kernel
+        MIN_SPACE_FOR_RAMFS_CREATION=167772160
+        lib_module_size_factor=1
+	# 250MB(268435456 Bytes) minimum space is required for more than 1 kernel
+	[ $No_of_symvers -gt 1 ] && MIN_SPACE_FOR_RAMFS_CREATION=268435456;lib_module_size_factor=2
+	lib_module_folder="/lib/modules"
+        boot_folder="/boot"
+	root_partition=$(df $lib_module_folder | grep -v Used | awk '{ print $1}')
+        boot_partition=$(df $boot_folder | grep -v Used | awk '{ print $1}')
+
+	ramdisk_size_factor=1
+	[ $root_partition != $boot_partition ] && ramdisk_size_factor=2
+
+        root_part_avail_space=$(echo $(($(stat -f --format="%a*%S" $lib_module_folder))))
```

Comment:
You may want to use %f instead of %a because LIS install is always done by root user.